src/data.coffee

```diff
@@ -14,18 +14,18 @@ data = module.exports = {}
 
 
 data.REGION = {
-  bucegi:     {bbox: [25.36, 45.33, 25.58, 45.52], title: "Bucegi"}
-  capatanii:  {bbox: [24.00, 45.17, 24.19, 45.34], title: "Căpățânii"}
-  ceahlau:    {bbox: [25.85, 46.84, 26.08, 47.04], title: "Ceahlău"}
-  ciucas:     {bbox: [25.84, 45.43, 26.05, 45.56], title: "Ciucaș"}
-  crai:       {bbox: [25.15, 45.49, 25.30, 45.58], title: "Piatra Craiului"}
-  fagaras:    {bbox: [24.30, 45.47, 24.89, 45.73], title: "Făgăraș"}
-  iezer:      {bbox: [24.85, 45.38, 25.10, 45.55], title: "Iezer"}
-  macin:      {bbox: [28.13, 45.07, 28.42, 45.29], title: "Măcin"}
-  nerei:      {bbox: [21.70, 44.80, 22.00, 45.03], title: "Cheile Nerei"}
-  parang:     {bbox: [23.43, 45.31, 23.82, 45.41], title: "Parâng"}
-  piatramare: {bbox: [25.59, 45.48, 25.75, 45.60], title: "Piatra Mare"}
-  retezat:    {bbox: [22.72, 45.29, 23.00, 45.42], title: "Retezat"}
+  bucegi:     {bbox: [25.36, 45.33, 25.58, 45.52], title: "Bucegi", meteoLink: "https://www.meteoblue.com/en/weather/forecast/week/mun%C5%A3ii-bucegi_romania_683598"}
```

Comment:
Sure, that'll work even if we have something like  `https://www.meteoblue.com/en/weather/forecast/week/44.946N21.863E`. Will do